tests/test_lib.py

```diff
@@ -1,53 +1,59 @@
 import sys
 sys.path = [".."] + sys.path
 
+import os
 import decimal
 
-from . import TEST_DIR
-from . import ccmtest
-from ccmlib.cluster import Cluster
+from tests import TEST_DIR
+from tests import ccmtest
+# from ccmlib.cluster import Cluster
+from ccmlib.scylla_cluster import ScyllaCluster
 import ccmlib
+# from ccmlib
 from six import StringIO
 
 CLUSTER_PATH = TEST_DIR
+if not os.path.exists(CLUSTER_PATH):
```

Comment:
let move this kind of code to `setUpClass` or to pytest session scoped fixture.
doing thing like that on module loading, it can't be problematic